bundle/Form/Type/FieldType/UserUpdateType.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Netgen\Bundle\InformationCollectionBundle\Form\Type\FieldType;
+
+use Symfony\Component\Form\Extension\Core\Type\EmailType;
+use Symfony\Component\Form\Extension\Core\Type\PasswordType;
+use Symfony\Component\Form\Extension\Core\Type\RepeatedType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\Validator\Constraints;
+
+final class UserUpdateType extends UserType
```

Comment:
We won't need this for info collection, please remove it.

Reply:
solved in 7822313d